ROADMAP.md

```diff
@@ -0,0 +1,35 @@
+# Power Platform Terraform Provider Roadmap
+
+This roadmap outlines the planned direction for the Power Platform Terraform Provider over the next 12 months. It serves as a high-level guide for contributors and users, providing transparency into where the project is headed while maintaining flexibility to adapt to community needs and evolving requirements.
+
+## Key Themes and Planned Direction
+
+| Key Theme | Planned Direction (12-month horizon) |
+|-----------|--------------------------------------|
+| **Quality & Reliability** | Increase test coverage across all resources and data sources, reduce flaky acceptance tests through improved test isolation and mock strategies, and formalize a predictable release cadence with automated quality gates. Focus on stability improvements for existing resources before adding new functionality. |
```

Comment:
Quality and Reliability is really about responding to community reported issues.  you can pull the last bullet into this one.  This is not so much about improving our coverage/testing process.  More about fixing reported issues.